dll/3rdparty/libtirpc/man/rpc.3t

```diff
@@ -222,7 +222,7 @@ struct svc_req {
     rpcvers_t    rq_vers;    /* service protocol version */
     rpcproc_t    rq_proc;    /* the desired procedure */
     struct opaque_auth    rq_cred;    /* raw creds from the wire */
-    caddr_t    rq_clntcred;    /* read only cooked cred */
+    caddr_t    rq_clntcred;    /* read-only cooked cred */
```

Comment:
This is 3rd-party code, please send the fix upstream.

https://git.reactos.org/?p=reactos.git;a=blob;f=media/doc/3rd+Party+Files.txt;hb=50774f865c4f9d9b926f32a8bcfc20bbeced8fb5#l97